core/src/filed/dir_cmd.cc

```diff
@@ -398,9 +398,12 @@ JobControlRecord* create_new_director_session(BareosSocket* dir)
   return jcr;
 }
 
-void* process_director_commands(void* p_jcr)
+void* process_initiated_director_commands(void* p_jcr)
```

Comment:
```suggestion
void* process_incoming_director_commands(void* p_jcr)
```

Reply:
Does this name fit to what the function does?

Reply:
hmm... technically yes but director commands are always incoming (even in the normal other process_directory_commands function).  This was supposed to distinguish between commands coming from a director that opened the connection vs a director where the fd initiated the connection.

Reply:
Im not 100% happy with the name either but i could not think of a better name. Maybe one could call it
```
process_director_commands_after_initiating_connection
```
but that also does not fit 100%.

Reply:
> hmm... technically yes but director commands are always incoming (even in the normal other process_directory_commands function). This was supposed to distinguish between commands coming from a director that opened the connection vs a director where the fd initiated the connection.

How about `process_fd_initiated_director_commands` ?

Reply:
sounds good to me!